test/functional/bulk.test.js

```diff
@@ -1892,7 +1892,7 @@ describe('Bulk', function () {
         expect(result).to.exist;
 
         bulk.execute(err => {
-          expect(err).to.match(/Batch cannot be re-executed/);
+          expect(err).to.match(/This batch has already been executed, create new batch to execute/);
```

Comment:
I think we should change this to an instanceOf check for MongoBatchReExecutionError

Reply:
Should I only change this one here and have a later PR handle changing all the error checks in the tests be instanceOf checks?

Reply:
This is the only one that was testing the message but it is preferred and its our API that its the instanceof that determines the error